Standards/scs-0219-v1-kaas-networking.md

```diff
@@ -0,0 +1,99 @@
+---
+title: KaaS Networking Standard
+type: Standard
+status: Draft
+track: KaaS
+---
+
+## Introduction
+
+Kubernetes defines a networking model that needs to be implented by a separate CNI plugin.
+Beyond basic connectivity within the cluster, however, there are many networking features that are specified but optional.
+Some of these optional features provide vital functionality, such as the NetworkPolicy API and the Ingress API.
+
+This standard specifies a minimal set of networking features that users can expect in clusters created by an SCS complient KaaS provider.
+
+## Terminology
+
+The following terms are used throughout this document:
+
+| Term | Meaning |
+|------|---------|
+| KaaS, managed Kubernetes | Kubernetes as a Service, automated on-demand deployment of Kubernetes clusters. |
+| CSP | Cloud Service Provider, the provider of the KaaS infrastructure. |
+| CNI | Container Network Interface, a standardized networking interface for container runtimes. |
+| CNI plugin, networking plugin | Kubernetes bindings for a CNI implementation, translates Kubernetes API concepts into more basic container networking concepts. |
+| network policy | A set of rules to restrict network traffic in a Kubernetes cluster. |
+
+## Motivation
+
+KaaS providers will typically support aditional networking functionality beyond basic Kubernetes networking.
+The specific range features depends on the used CNI plugin, but may also be extended by additional operators.
+Users may expect certain optional functionality, so we should define a baseline feature set that has to be available in an SCS-compliant KaaS cluster.
+
+## Design Considerations
+
+The Kubernetes API is arbitrary extensible.
+Many CNI plugins will define custom resources to enable functionality that is not covered in the official [API specification](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.31/).
+Sometimes they will even reuse names from different API groups, such as `NetworkPolicy`, which exists in the basic `networking.k8s.io/v1` API, but also in `projectcalico.org/v3`.
+
+To avoid any ambiguity, we should therefore be explicit about the API groups and versions of resources.
+We should also avoid mandating third party API extensions, to avoid dependencies on specific third party software and keep the standard as generic as possible.
```

Comment:
```suggestion
We should also avoid mandating third-party API extensions, to avoid dependencies on specific third-party software and keep the standard as generic as possible.
```